tests/integration/api.test.js

```diff
@@ -241,7 +299,22 @@ describe.skip('#getBids', () => {
   })
 })
 
+test('#getCreateAccountAuthorization', async () => {
+  CoordinatorAPI.setBaseApiUrl(mockUrlAPI)
+
+  const hermezEthereumAddress = 'hez:0x74a549b410d01d9eC56346aE52b8550515B283b2'
+  const bjjAddress = 'hez:dEZ-Tj7d5h0TAqbnRTTYURYDEo5KZzB87_2WknUU8gCN'
+  const signature = '0x8db6db2ad6cbe21297fb8ee01c59b01b52d4df7ea92a0f0dee0be0075a8f224a06b367407c8f402cfe0490c142a1c92da3fc29b51162ae160d35e1577d3071bb01'
+
+  const res = await CoordinatorAPI.getCreateAccountAuthorization(hermezEthereumAddress)
+  expect(res.hezEthereumAddress).toBe(hermezEthereumAddress)
+  expect(res.bjj).toBe(bjjAddress)
+  expect(res.signature).toBe(signature)
+})
+
 test('#postCreateAccountAuthorization', async () => {
+  CoordinatorAPI.setBaseApiUrl(mockUrlAPI)
+
   const privateKeyEth = '0x0000000000000000000000000000000000000000000000000000000000000001'
   const signer = { type: 'WALLET', privateKey: privateKeyEth }
   const { hermezWallet } = await createWalletFromEtherAccount('http://localhost:8545', signer)
```

Comment:
There is a similar test in unit testing checking `signCreateAccountAuthorization` in  https://github.com/hermeznetwork/hermezjs/blob/feature/setup-tests/tests/unit/hermez-wallet.test.js#L95

I would like to change this `mockUrlAPI` by a `realAPI` (integration-testing environment) but it would mean that some previous actions would be required before checking the API.